lib/github_api/client/orgs.rb

```diff
@@ -5,6 +5,7 @@ module Github
   class Client::Orgs < API
 
     require_all 'github_api/client/orgs',
+                'hooks'
```

Comment:
@pkhxcp you are missing a comma here to separate the arguments out, hence the tests failure